app/util/supabase/supabase.server.ts

```diff
@@ -0,0 +1,41 @@
+import { createCookieSessionStorage } from "@remix-run/node";
+import { createServerClient, parse, serialize } from "@supabase/ssr";
+import { type Session } from "@supabase/supabase-js";
+import { Database } from "database.types";
+
+export function createSupabaseServerClient(request: Request) {
+  const cookies = parse(request.headers.get("Cookie") ?? "");
+  console.log("Cookies in createSupabaseServerClient", cookies);
```

Comment:
best practice not to log cookies unless in a dev env

Reply:
I went through and removed some other logs too.